redux/main.py

```diff
@@ -91,11 +91,11 @@ def __init__(
         self._event_handlers_queue = queue.Queue[
             tuple[EventHandler[Event], Event] | None
         ]()
-        self_workers = [
+        self.workers = [
```

Comment:
I meant this:
```suggestion
        workers = [
```
It should be either `self._workers` or `workers`, I prefer local variable `workers`

Reply:
ok lets keep local variable `workers`